src/main/java/org/apache/taglibs/standard/util/XmlUtil.java

```diff
@@ -69,7 +69,14 @@
     private static final DocumentBuilderFactory PARSER_FACTORY;
     private static final SAXTransformerFactory TRANSFORMER_FACTORY;
     private static final SAXParserFactory SAXPARSER_FACTORY;
+
+    private static boolean secureProcessing;
+
     static {
+
+        String secure = System.getProperty("org.apache.taglibs.secureXmlProcessing");
+        secureProcessing = ! (secure != null && secure.equalsIgnoreCase("false"));
```

Comment:
Well you are right, default should be true, but the code construct in general is simpler this way.
this method is present on java.lang.Boolean since ever (java v 1.0.2).
see http://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#getBoolean%28java.lang.String%29